File: Python3/Module-1/gcd.py
"""
This file is to find the GCD of given two inputs
"""
def GCD(m,n):
    if m<n:
        temp = m
        m = n
        n = temp
    temp = m%n
    while temp != 0:
        m = n
        n = temp
        temp = m%n
    return n

def main():
    print("Give inputs m,n: ")
    m,n = [int(i) for i in input().split(" ")]
    print(GCD(m,n))

if "__name__"="__main__":
    main()

File: C-Programming/Module-1/gcd.c
// find the GCD of given two numbers
// using Eucldian Algo

#include <stdio.h>

int main(){
  int m,n,temp;
  printf("Enter the value of m and n: ");
  scanf("%d%d", &m,&n);
  if(n>m){
    temp = m;
    m = n;
    n = temp;
  }
  printf("m=%d and n=%d", m,n);
  temp = m%n;
  while (temp!=0) {
    m = n;
    n = temp;
    temp = m%n;
  }
  printf("\nGCD is: %d",n);

  return 0;
}
